.github/workflows/test_suite.yml

```diff
@@ -0,0 +1,29 @@
+name: Test suite
+
+on: [push, pull_request]
+
+jobs:
+  build-linux:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x]
+        arch: [x64]
+        
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Install neu & link it
+      working-directory: ./
+      run: |
+        npm install --global @neutralinojs/neu
```

Comment:
Could you please remove this line since we are testing the current code with `npm link`.. Thanks

Reply:
Done !